src/commands/mdn.ts

```diff
@@ -89,26 +82,39 @@ const Mdn: Command = {
 };
 export default Mdn;
 
-const BASE_URL = `https://www.googleapis.com/customsearch/v1/siterestrict?key=${process.env.CSE_KEY}&cx=${process.env.CSE_CSX}`;
+async function refreshIndex() {
+	const res = await fetch(MDN_INDEX);
+	if (res.ok) {
+		index = await res.json();
+		index.forEach(({ title }, id) => searcher.add(id, title));
+	}
+}
+refreshIndex();
+const REFRESH_INTERVAL = Number(process.env.MDN_INDEX_REFRESH) || 12; // hours
```

Comment:
Number() shouldn't be necessary here I believe,because the env validator should have already coerced it.

Reply:
Yes but the linter still complains